components/catalogs/front/src/components/ProductMapping/hooks/useSystemAttributesFactories.ts

```diff
@@ -0,0 +1,21 @@
+import {useMemo} from 'react';
+import {useTranslate} from '@akeneo-pim-community/shared';
+import {Attribute} from '../../../models/Attribute';
+
+export const useSystemAttributesFactories = (): Attribute[] => {
```

Comment:
```suggestion
export const useSystemAttributes = (): Attribute[] => {
```
Also, there might be some refactoring to be done to have a more robust type-hinting, (but not in this task)

Reply:
@RiffVentura Factories is the term used in ProductSelection. It's judicious to stay coherent, are you agree ?